api/v1alpha1/validation.go

```diff
@@ -0,0 +1,41 @@
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/validation"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+// ValidateLabels validates that a set of labels are correctly defined.
+func ValidateLabels(labels *map[string]string) error {
```

Comment:
Please either link to JIRA [MGMT-17069](https://issues.redhat.com/browse/MGMT-17069) or close it as a duplicate

Reply:
Done